From 296e5f96c2bc5092cd7ace19b71b347be5c7a1b1 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Thu, 7 Feb 2019 11:54:48 -0600
Subject: [PATCH 5/5] common: Label livedisplay 2.0 sysfs service

* Add label for acl node too for devices that use it

Change-Id: Iaa69ad07f122a6a74182afe7db49d4c21bd88842
---
 common/private/genfs_contexts | 1 +
 common/vendor/file_contexts   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/common/private/genfs_contexts b/common/private/genfs_contexts
index c9c219c..62ba3e3 100644
--- a/common/private/genfs_contexts
+++ b/common/private/genfs_contexts
@@ -1,6 +1,7 @@
 genfscon fuseblk / u:object_r:vfat:s0
 genfscon sdfat / u:object_r:exfat:s0
 
+genfscon sysfs /devices/virtual/graphics/fb0/acl u:object_r:sysfs_livedisplay_tuneable:s0
 genfscon sysfs /devices/virtual/graphics/fb0/aco u:object_r:sysfs_livedisplay_tuneable:s0
 genfscon sysfs /devices/virtual/graphics/fb0/cabc u:object_r:sysfs_livedisplay_tuneable:s0
 genfscon sysfs /devices/virtual/graphics/fb0/color_enhance u:object_r:sysfs_livedisplay_tuneable:s0
diff --git a/common/vendor/file_contexts b/common/vendor/file_contexts
index 89b5264..508c63f 100644
--- a/common/vendor/file_contexts
+++ b/common/vendor/file_contexts
@@ -7,6 +7,7 @@
 # LiveDisplay HAL
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-legacymm u:object_r:hal_lineage_livedisplay_default_exec:s0
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-sdm u:object_r:hal_lineage_livedisplay_default_exec:s0
+/(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-sysfs u:object_r:hal_lineage_livedisplay_default_exec:s0
 
 # Trust HAL
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.trust@1\.0-service u:object_r:hal_lineage_trust_default_exec:s0
-- 
2.17.1

